contracts/dapis/DapiServer.sol

```diff
@@ -75,13 +77,17 @@ contract DapiServer is
     mapping(bytes32 => DataFeed) public override dataFeeds;
 
     /// @notice Data feed with ID specific to the OEV proxy
+    /// @dev This implies that an update as a result of an OEV auction only
+    /// affects contracts that read through the respective proxy that the
+    /// auction was being held for
```

Comment:
I'm just checking if I understood this correctly. So this is saying that the update that resulted from an OEV auction being won will only change the data feed value that oev readers read. Not the value of the data feed that other readers read. (because oev readers read from this mapping and others like byog read directly from dataFeeds mapping)

Reply:
> So this is saying that the update that resulted from an OEV auction being won will only change the data feed value that oev readers read

So if project X is using ETH/USD with OEV, they have a special proxy that they read from. We hold an auction specific to that proxy address, and the related OEV updates only affect this proxy (or readers that pretend to be this proxy). Project Y using ETH/USD with OEV is not affected.

> others like byog read directly from dataFeeds mapping

I assume you mean self-funded data feeds and not byog the data provider. Self-funded data feeds will also be read through proxies, which optionally support OEV.